### PR TITLE
show all users and owner user

### PR DESCRIPTION
**Pull request by hiennv94**

### CODE REVIEW

File: app/assets/javascripts/users.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (dieunb) [inline app/assets/javascripts/users.coffee]: remove this file

File: app/assets/stylesheets/users.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Users controller here.
```
Comment (dieunb) [inline app/assets/stylesheets/users.scss]: remove this file

File: app/helpers/users_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module UsersHelper
```
Comment (dieunb) [inline app/helpers/users_helper.rb]: remove this file if you don't use it

File: app/views/users/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<% provide :title, t("application.general.allUser") %>
+<h1><%= t("application.general.showall") %></h1>
+
+<%= will_paginate %>
+    <div class="col-md-12">
```
Comment (dieunb) [inline app/views/users/index.html.erb]: indent

File: app/views/users/show.html.erb [hunk 5]
```diff
@@ -0,0 +1,21 @@
+<% provide :title, t("application.general.showProfile") %>
+<div class="container">
+  <div class="row information">
+    <div class="row1">
+      <div id="showName" class="col-md-9">
+        <div class="col-md-3">
+          <strong>Name</strong>
```
Comment (dieunb) [inline app/views/users/show.html.erb]: i18n

File: app/views/users/show.html.erb [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+        </div>
+        <div class="col-md-6"><strong><span><%= @user.name %></span></strong></div>
+      </div>
+    </div>
+    <div class="row2">
+      <div class="col-md-9">
+        <div class="col-md-3">
+          <strong>Email</strong>
```
Comment (dieunb) [inline app/views/users/show.html.erb]: i18n

File: config/routes.rb [hunk 7]
```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
+  get 'users/index'
```
Comment (dieunb) [inline config/routes.rb]: double quote pls

File: config/routes.rb [hunk 8]
```diff
@@ -1,4 +1,8 @@
...
+
+  get 'users/show'
```
Comment (dieunb) [inline config/routes.rb]: double quote pls

File: config/routes.rb [hunk 9]
```diff
@@ -1,4 +1,8 @@
...
+
```
Comment (dieunb) [inline config/routes.rb]: why don't use `resources :users`

**Comment by truonglocbinh** [comment]
OK

**Pull request merged**